editor.planx.uk/src/pages/FlowEditor/lib/__tests__/flags.test.ts

```diff
@@ -1,213 +0,0 @@
-import { ComponentType as TYPES } from "@opensystemslab/planx-core/types";
```

Comment:
This outdated test suite is removed in favor of new/updated files: 
- `filters.test.ts`
- `preview/autoAnswerableFlag.test.ts`